src/index.js

```diff
@@ -101,25 +90,55 @@ const addChecktext = () => {
   }
 };
 
+const deleteComplete = () => {
+  const clearComplete = document.querySelector('.clear-complete');
+  if (clearComplete && todo.check) {
+    clearComplete.addEventListener('click', () => {
+      todo.check.sort();
+      for (let k = todo.check.length - 1; k >= 0; k -= 1) {
+        todo.taskArr.splice(todo.check[k], 1);
+      }
+      for (let j = 0; j < todo.taskArr.length; j += 1) {
+        todo.taskArr[j].index = j + 1;
+      }
+      todo.storeLocalStorage();
+      window.location.reload();
+    });
+  }
+};
+
```

Comment:
- [ ] kindly consider making sure that the Array.filter() method is used on this function to meet this project requirement 
> Implement a function for the "Clear all completed" button (use filter() method).

Reply:
I have applied the filter method to update the array as per instructions.